packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -515,7 +517,7 @@ export class InputManager {
                 scene.onPointerUp(evt, pickResult, PointerEventTypes.POINTERUP);
             }
 
-            if (!clickInfo.hasSwiped && !this._skipPointerTap) {
+            if (!clickInfo.hasSwiped && !this._skipPointerTap && !this._isMultiTouchGesture) {

```

Comment:
I wonder if swiped is not inside the multitouch case already ???

Reply:
Well, the swiped state comes from taking the `_startingPointerPosition` and current pointer position and checking that the differences is greater than the `DragMovementThreshold`.  Since we only have a single instance of `_startingPointerPosition`, that value will get clobbered by either touch's movement.  The fix for this is to either expand `_startingPointerPosition` to track all active pointers OR ignore the swiping state and don't fire any taps when multiple pointers are active.  Since the swiping state is only used to check for TAP or DOUBLETAP, we should be able to just use the second option as any form of tapping shouldn't be allowed during a multi-touch gesture.

Reply:
Okay, updated!